### PR TITLE
Revive test coverage #1140

### PR DESCRIPTION
**Pull request by lzxm160**
work on #1140 

Merging this into master will increase coverage by 2.56%

### CODE REVIEW

File: blockchain/actionpicker_test.go [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/iotexproject/iotex-core/action"
+	"github.com/iotexproject/iotex-core/actpool/actioniterator"
+	"github.com/iotexproject/iotex-core/test/identityset"
+)
+
+func TestPickAction(t *testing.T) {
```
Comment (zjshen14) [inline blockchain/actionpicker_test.go]: The effectiveness of this test should be testing against the gas limit

File: server/itx/server_test.go [hunk 2]
```diff
@@ -0,0 +1,106 @@
+// Copyright (c) 2019 IoTeX
```
Comment (zjshen14) [inline server/itx/server_test.go]: I feel these tests are not so useful, as it's a integration place. Instead, we need to write more e2e integration tests

File: server/itx/heartbeat_test.go [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+	s, err := NewServer(cfg)
+	cfg.Consensus.Scheme = config.RollDPoSScheme
+	cfg.Genesis.EnableGravityChainVoting = true
+	require.NoError(err)
+	require.NotNil(s)
+	handler := NewHeartbeatHandler(s)
+	require.NotNil(handler)
+	require.Panics(func() { handler.Log() }, "P2pAgent is nil")
+
+	ctx, cancel := context.WithCancel(context.Background())
```
Comment (zjshen14) [inline server/itx/heartbeat_test.go]: What does the following try to test?

File: server/itx/subchain_test.go [hunk 4]
```diff
@@ -7,10 +7,20 @@
 package itx
```
Comment (zjshen14) [inline server/itx/subchain_test.go]: You could ignore subchain related, which might be deprecated.

File: server/itx/heartbeat_test.go (hunk 3 above)
Comment (lzxm160) [reply server/itx/heartbeat_test.go]: test HeartbeatHandler.Log,but there's no returned value

File: server/itx/subchain_test.go (hunk 4 above)
Comment (lzxm160) [reply server/itx/subchain_test.go]: sure

**Pull request merged**